src/Fmt_ast.ml

```diff
@@ -2667,30 +2666,32 @@ and fmt_type_declaration c ?(pre = "") ?(suf = ("" : _ format)) ?(brk = suf)
   in
   update_config_maybe_disabled c ptype_loc ptype_attributes
   @@ fun c ->
+  let break_before_manifest_kind =
```

Comment:
You could move this inside `fmt_manifest`

Reply:
done